### ISSUE TITLE
[Feature Request] Avoid Fading on Enemies

### ISSUE THREAD

**Issue by Dahgoth** [kesava-wow/kuinameplates2#190]
I'll try to explain what I mean via screenshots:
![wowscrnshot_032017_223234](https://cloud.githubusercontent.com/assets/1629256/24137882/cc6e08ea-0e26-11e7-8fa0-e63dbd422443.jpg)
As of now it is possible to see nameplates on both in and out of combat enemies but when you target something all other nameplates fading (I have set alpha to 0 to not see non-tracked NPCs in towns and order halls).
![wowscrnshot_032017_223237](https://cloud.githubusercontent.com/assets/1629256/24137930/1335fc74-0e27-11e7-9e5e-4eb48fc1469c.jpg)
I know that I can track enemies to prevent it but it is usefull to not do that so I can see which enemies are tracked for quests and which are not:
![wowscrnshot_022317_175943](https://cloud.githubusercontent.com/assets/1629256/24137985/619bcec0-0e27-11e7-84b0-55fc1ab54248.jpg)

TL:DR it should look like this:
- Enemies OoC with you - healthbar
- Enemies IC with you - healthbar and text
- Enemies tracked for quests (in blizz options) - healthbar and text
- Friendly NPCs tracked for quests (in blizz options) - text
- Friendly NPCs not tracked for quests (in blizz options) - nothing

I think it will be enough to have an option to avoid fading on enemies =)

**Comment by kesava-wow** [comment]
As I already have vague plans to change the way fading behaves (https://github.com/kesava-wow/kuinameplates2/issues/75), try using this: https://github.com/kesava-wow/kuinameplates-customs/blob/master/custom.avoid-fade-hostile.lua

Read here for how to install it: https://github.com/kesava-wow/kuinameplates-customs

After installing it, hostile nameplates shouldn't fade out at all.

**Comment by Dahgoth** [comment]
Thanks a lot! Love your amazing addon!
